features/cli/oc_import_image.feature

```diff
@@ -108,13 +108,13 @@ Feature: oc import-image related feature
     When I run the :get client command with:
       | resource        | imagestreams |
     Then the output should match:
-      | .*deployment-example.* |
+      | .*deployment-example.* | 
```

Comment:
Looks like unexpected change ?

Reply:
Change it by accident, but no idea why it cannot come back

Reply:
Did you delete the white spaces at end of line and add it to the commit ?

Reply:
Fixed.